lib/mayor_game/city/buildable.ex

```diff
@@ -280,6 +282,22 @@ defmodule MayorGame.City.Buildable do
           pollution: 10
         }
       },
+      # Megablocks ————————————————————————————————————
+      megablocks: %BuildableMetadata{
+        level: 0,
```

Comment:
I just added a field for "category" to these structs; could you add `category: :housing,` to this here? ty!

Reply:
Done